rdf-delta-base/pom.xml

```diff
@@ -45,6 +45,10 @@
       <artifactId>slf4j-jdk14</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
FYI: This would pull in the whole of Fuseki into the client library including a copy of Eclipse jetty.